### PR TITLE
reverting workflow changes

### PR DESCRIPTION
**Pull request by prayanshchh**
**What kind of change does this PR introduce?**
reverting workflow changes

**Issue Number:**

Fixes #

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

## Summary by CodeRabbit

- **New Features**
	- Introduced new jobs to the workflow: `Start-App-Without-Docker` and `Docker-Start-Check` for enhanced app startup verification.
- **Improvements**
	- Enhanced checks for sensitive files by adding more file patterns.
	- Improved coverage reporting with a dynamic naming convention for the upload step.
	- Minor adjustments to environment variable references for better accuracy.

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -123,6 +124,12 @@ jobs:
             ISSUE_GUIDELINES.md
             PR_GUIDELINES.md
             README.md
+            *.pem
+            *.key
```
Comment (palisadoes) [inline .github/workflows/pull-request.yml:128]: Why were these added? Why would we be storing cryptography and cybersecurity related files in the repository?

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -272,6 +279,126 @@ jobs:
...
+            sleep 1
+            timeout=$((timeout-1))
+            if [ $((timeout % 10)) -eq 0 ]; then
+              echo "Still waiting for production app to start... ${timeout}s remaining"
+            fi
+          done
+          if [ $timeout -eq 0 ]; then
+            echo "Timeout waiting for production application to start"
+            exit 1
+          fi
```
[the remark this answers is not in the document]
Comment (prayanshchh) [reply .github/workflows/pull-request.yml:320]: @@coderabbitai all the workflows aren't getting trigerred, check pull-request.yml file

File: .github/workflows/pull-request.yml [hunk 3]
```diff
@@ -272,6 +279,126 @@ jobs:
 
       - name: Validate Documents
         run: graphql-inspector validate './src/GraphQl/**/*.ts' './talawa-api/schema.graphql'
+    Start-App-Without-Docker:
+    name: Check if Talawa Admin app starts (No Docker)
```
Comment (palisadoes) [inline .github/workflows/pull-request.yml:283]: Incorrect indentation

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (prayanshchh) [reply .github/workflows/pull-request.yml:128]: it was present in pull-request.yml file, this changes was reverted

Comment (palisadoes) [reply .github/workflows/pull-request.yml:128]: OK. They are not necessary, but let them remain.

**Comment by prayanshchh** [comment]
please check  @palisadoes @varshith257

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (prayanshchh) [reply .github/workflows/pull-request.yml:128]: alright sir

File: .github/workflows/pull-request.yml (hunk 3 above)
Comment (prayanshchh) [reply .github/workflows/pull-request.yml:283]: done

**Pull request merged**